server/plugin/command_test.go

```diff
@@ -197,3 +197,45 @@ func TestParseCommand(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckConflictingFeatures(t *testing.T) {
+	type output struct {
+		valid               bool
+		conflictingFeatures []string
+	}
+	tests := []struct {
+		name string
+		args []string
+		want output
+	}{
+		{
+			name: "no conflicts",
+			args: []string{"creates", "pushes", "issue_comments"},
+			want: output{true, nil},
+		},
+		{
+			name: "conflict with issue and issue creation",
+			args: []string{"pulls", "issues", "issue_creations"},
+			want: output{false, []string{"issues", "issue_creations"}},
+		},
+		{
+			name: "conflict with pulls and pulls created",
+			args: []string{"pulls", "issues", "pulls_created"},
+			want: output{false, []string{"pulls", "pulls_created"}},
+		},
+		{
+			name: "conflict with pulls and pulls merged",
+			args: []string{"pulls", "pushes", "pulls_merged"},
+			want: output{false, []string{"pulls", "pulls_merged"}},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ok, fs := checkFeatureConflict(tt.args)
+			got := output{ok, fs}
+			errMsg := fmt.Sprintf("checkFeatureConflict() = %v, want %v", got, tt.want)
+			assert.EqualValues(t, tt.want, got, errMsg)
+		})
+	}
+}
```

Comment:
Can we have one successful test case as well?

Reply:
Hi @mickmister.

Thank you for your comment.

Could you please clarify what do you mean?
As far as I can see, this test case demonstrates a positive scenario, does not it?
https://github.com/mattermost/mattermost-plugin-github/pull/712/files/a82fe13ef9f7433b2e9a6ac9acb2f72f27dbac74#diff-6aaf76d180fbe90f62e3da82e7d832fa519ce20840b4d5035fcecd250cc76975R211-R215

Reply:
Thanks for clarifying. I had misread the test :+1: It's a bit difficult to understand since we have a line that says `errMsg` that is always defined (even in the case when an error did not occur), followed by an assertion that at a glance I assumed was on that message's contents. Maybe we can name the variable something like `testFailureMessage`?

Reply:
Thanks for your suggestion. Renamed.